docs/src/pages/components/data-grid/editing/editing.md

```diff
@@ -7,6 +7,33 @@ components: DataGrid, XGrid
 
 <p class="description">Data Grid will provide full support to create, read, update, and delete operations (CRUD).</p>
 
+## Cell editing
+
+Cell editing allows to edit the value of one cell at the time.
+To activate editing on cells, set the 'editable' property of the column definition `GridColDef` object to true.
+
+```tsx
+<DataGrid columns={[{ field: 'name', editable: true }]} />
+```
+
+By default, a cell can turn into edit mode by double clicking on it, or using <kbd>ENTER</kbd> key, if the cell has focus.
+
+{{"demo": "pages/components/data-grid/editing/BasicEditingGrid.js", "bg": "inline"}}
```

Comment:
<img width="499" alt="Screenshot 2021-03-24 at 20 20 03" src="https://user-images.githubusercontent.com/3165635/112371031-6d575380-8cde-11eb-90af-26b734d134e2.png">

Seems to add nothing, we could hide it.

```suggestion
{{"demo": "pages/components/data-grid/editing/BasicEditingGrid.js", "bg": "inline", "defaultCodeOpen": true}}
```